staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -151,11 +212,100 @@ func WithTypeAndObjectMeta(s *schema.Structural) *schema.Structural {
 	props["metadata"] = schema.Structural{
 		Generic: schema.Generic{Type: "object"},
 		Properties: map[string]schema.Structural{
-			"name": stringType,
+			"name":         stringType,
 			"generateName": stringType,
 		},
 	}
 	result.Properties = props
 
 	return result
-}
\ No newline at end of file
+}
+
+// estimateMinSizeJSON estimates the minimum size in bytes of the given schema when serialized in JSON.
+// minLength/minProperties/minItems is not currently taken into account, so the estimate is a little
+// smaller than the actual smallest size.
+func estimateMinSizeJSON(s *schema.Structural) int64 {
+	switch s.Type {
+	// take a comma into account for all cases since we'll generally call this function for arrays/maps
+	case "boolean":
+		// true,
+		return 5
+	case "number", "integer":
+		// 0,
+		return 2
+	case "string":
+		if s.ValueValidation != nil && s.ValueValidation.Format != "" && s.ValueValidation.Format != "byte" {
```

Comment:
Same as previous comment` s.ValueValidation.Format != "" && s.ValueValidation.Format != "byte"` checks are not needed

Reply:
Right, I think the initial line of thought behind the `byte` clause was to prevent the switch statement from being executed in that case, since we don't do any special checks when `Format` is set to `byte` After taking another look, I think you're right, since we'll just fall through to `return 3` if `Format` is set so. I'll change it.